### DISCUSSION TITLE
Advice on topic modelling?

### DISCUSSION THREAD

**Discussion by amhanson9** [Digital-Preservation-Coalition/PythonStudyGroups#30] [General]
I'm doing an experiment with topic modelling to see if it helps with determining common subjects in born-digital archival collections (e.g., when someone donates their computer to our special collections) to use for description. This is a brand-new area of Python for me and not something anyone in my group has done, so I'm wondering if anyone else has advice?

So far, I can get the text out of a variety of formats using the tika library and I've worked through a few tutorials on topic modeling from the YouTube channel [Python Tutorials for Digital Humanities](https://www.youtube.com/playlist?list=PL2VXyKi-KpYttggRATQVmgFcQst3z6OlX). They cover scikit-learn, gensim, spacy, and top2vec. Anyone have luck with any of those, or prefer something else?

**Comment by mutanthumb** [comment]
Hi Adriane-
I have not done any Topic Modeling, I found this: https://medium.com/nlplanet/text-analysis-topic-modelling-with-spacy-gensim-4cd92ef06e06 which mentions similar modules as you do above. I put a question out Mastodon to see if anyone has any advice.
-susan

**Reply by amhanson9** [reply]
Thank you, I appreciate you taking a look!

**Comment by eli-hu** [comment]
Following! Thank you for the link to the Python Tutorials for DH. I'll take a look at those.
